pkgs/development/tools/libsigrok/default.nix

```diff
@@ -1,8 +1,23 @@
-{ lib, stdenv, fetchurl, pkg-config, libzip, glib, libusb1, libftdi1, check
-, libserialport, librevisa, doxygen, glibmm, python
-, version ? "0.5.1", sha256 ? "171b553dir5gn6w4f7n37waqk62nq2kf1jykx4ifjacdz5xdw3z4", doInstallCheck ? true
+{ lib
+, stdenv
+, fetchurl
+, pkg-config
+, libzip
+, glib
+, libusb1
+, libftdi1
+, check
+, libserialport
+, librevisa
+, doxygen
+, glibmm
+, python
+, hidapi
+, sigrok-firmware-fx2lafw
+, version ? "0.5.2"
```

Comment:
Consider moving version and sha to the proper location with the override removed as well. And tbh I don't understand why this was done like it was in the first place...

Reply:
And doInstallCheck too

Reply:
Done. I moved the commit removing the override/version pinning to the front, so that the order is a bit more consistent now.

While anyway making more changes, I enabled ieee1284 and bluetooth support on Linux.